File: app/src/main/java/me/kirimin/mitsumine/model/MyBookmark.java
package me.kirimin.mitsumine.model;

public class MyBookmark {

    private String title;
    private String comment;
    private int bookmarkCount;
    private String linkUrl;

    public MyBookmark(String title, String comment, int bookmarkCount, String linkUrl) {
        this.title = title;
        this.comment = comment;
        this.bookmarkCount = bookmarkCount;
        this.linkUrl = linkUrl;
    }

    public String getTitle() {
        return title;
    }

    public String getComment() {
        return comment;
    }

    public int getBookmarkCount() {
        return bookmarkCount;
    }

    public String getLinkUrl() {
        return linkUrl;
    }
}

File: app/src/main/java/me/kirimin/mitsumine/_common/domain/model/MyBookmark.kt
package me.kirimin.mitsumine._common.domain.model

import me.kirimin.mitsumine._common.network.entity.MyBookmarksResponse

class MyBookmark(private val response: MyBookmarksResponse.MyBookmarkResponse) {

    val title get() = response.entry.title
    val comment get() = response.comment
    val bookmarkCount get() = response.entry.count
    val linkUrl get() = response.entry.url
    val snippet: String get() = response.entry.snippet
}